lib/autoload/coursemology.rb

```diff
@@ -0,0 +1,7 @@
+module Coursemology
+  class ComponentNotFoundError < StandardError
```

Comment:
Seems like it's a convention to scope exceptions under project name ? Like 'Cancan::AccessDenied'